SparseGrids/tasgrid_main.cpp

```diff
@@ -156,6 +156,7 @@ int main(int argc, const char ** argv){
             if ((strcmp(argv[k],"global") == 0)) test = test_global;
             if ((strcmp(argv[k],"local") == 0)) test = test_local;
             if ((strcmp(argv[k],"wavelet") == 0)) test = test_wavelet;
+            if ((strcmp(argv[k],"fourier") == 0)) test = test_fourier;
```

Comment:
* looks much better
* you need to add a test to `SparseGrids/CMakeLists.txt` with cmake command `add_test()`
* then we can test on the CI

```
add_test(SparseGridsFourier      tasgrid -test fourier)
```

Reply:
Done